x-pack/plugins/secops/public/containers/events/index.tsx

```diff
@@ -11,11 +11,11 @@ import { pure } from 'recompose';
 
 import { EventItem, GetEventsQuery, KpiItem } from '../../../common/graphql/types';
 
-import { eventsQuery } from './events.gql_query';
+import { eventsQuery } from './index.gql_query';
 
 export interface EventsArgs {
-  events: EventItem[];
-  kpiEventType: KpiItem[];
+  events?: EventItem[];
+  kpiEventType?: KpiItem[];
```

Comment:
Are these really optional now? I still think we should be structuring our code and types to not have as many optionals (or any at all) so we don't end up abusing the `!` operator so much. When I was wiring up redux, I was nailed by several runtime errors and type script is not doing its job of type checking arguments I might not be sending in.

I also corrected at least 1 `?` optionals with `!` operator from a previous pull request that wasn't needed which is why I'm kind of on the look out now ;-)

Reply:
I just need to make sure that I changed the fetch policy to `cache-and-network`

Reply:
in case you need it in the future, it is always an important doc
https://www.apollographql.com/docs/react/api/react-apollo.html#graphql-config-options-fetchPolicy

Reply:
Just read through and Apollo's documentation from above, thanks! Apollo has great docs and a good API. 

We have two other areas after this PR left with `no-cache` which will be the simple 'who-am-i' and the `WithSources` used with hosts:
https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/secops/public/containers/source/index.tsx#L26

Nothing holding up this PR but pointing it out if you want fast load times from cache when you can first and then network for consistency second to be universal and you're just missing a spot or two.